service/src/main/java/uk/gov/hmcts/reform/fpl/utils/DateFormatterHelper.java

```diff
@@ -34,6 +34,11 @@ public static LocalDate parseLocalDateFromStringUsingFormat(String date, String
         return LocalDate.parse(date, DateTimeFormatter.ofPattern(format, Locale.UK));
     }
 
+    public static String formatLocalDateFromStringToStringUsingFormat(String date, String format) {
+        LocalDate localDate = LocalDate.parse(date);
+        return formatLocalDateToString(localDate, format);
+    }
+
```

Comment:
from string to string? why?

Reply:
Date format in OtherParty is  ""2019-01-01" but we have to display it in a different format as "1 January 2019"

Reply:
Removed it and moved the check to TemplateGenerationService